website/blog/authors.yml

```diff
@@ -1,379 +1,348 @@
+    - icon: fa-linkedin
+      url: https://www.linkedin.com/in/clrcrl/
+  name: Claire Carroll
+  organization: analyticsengineers.club
 dave_connors:
-  name: Dave Connors
-  job_title: Senior Analytics Engineer
-  organization: dbt Labs
   image_url: /img/blog/authors/dconnors.jpeg
+  job_title: Senior Analytics Engineer
   links:
-    - url: https://github.com/dave-connors-3
-      icon: fa-github
-    - url: https://www.linkedin.com/in/daveconnors3/
-      icon: fa-linkedin
-
-ross_turk:
-  name: Ross Turk
-  job_title: VP Marketing
-  organization: Datakin
-  image_url: /img/blog/authors/ross-turk.png
-  links:
-    - url: https://mobile.twitter.com/rossturk
-      icon: fa-twitter
-    - url: https://github.com/rossturk
-      icon: fa-github
-
-sung_chung:
-  name: Sung Won Chung
-  job_title: Solutions Architect
+    - icon: fa-github
+      url: https://github.com/dave-connors-3
+    - icon: fa-linkedin
+      url: https://www.linkedin.com/in/daveconnors3/
+  name: Dave Connors
   organization: dbt Labs
-  image_url: /img/blog/authors/sung.jpeg
-  links:
-    - url: https://www.linkedin.com/in/sungwonchung1/
-      icon: fa-linkedin
-
-seth_rosen:
-  name: Seth Rosen
-  job_title: Co-Founder & CEO
-  organization: TopCoat Data
-  description: Seth Rosen is co-founder and CEO of TopCoat Data - a platform for helping organizations build analytical applications. Prior to founding TopCoat, Seth helped companies of all sizes build custom data apps on top of the modern data stack through his consultancy, Hashpath. When he’s not tweeting and thinking about data, he’s tweeting while parenting two toddlers.
-  image_url: /img/blog/authors/seth-rosen.jpeg
+david_krevitt:
+  description: Laziness is a virtue
+  image_url: /img/blog/authors/dkrevitt.jpg
+  job_title: Marketing
   links:
-    - url: https://twitter.com/sethrosen
-      icon: fa-twitter
-    - url: https://www.linkedin.com/in/sdrosen/
-      icon: fa-linkedin
-
+    - icon: fa-twitter
+      url: https://twitter.com/dkrevitt
+  name: David Krevitt
+  organization: dbt Labs
 donny_flynn:
-  name: Donny Flynn
-  job_title: Customer Data Architect
-  organization: Census
   description: Donny Flynn is a customer data architect at Census - a reverse ETL product that helps companies operationalize analytics. Prior to joining Census, Donny built out the data stack at Owner and led a data team at Chiper. When not doing data work, he's most likely watching Chicago sports.
   image_url: /img/blog/authors/dflynn.jpeg
+  job_title: Customer Data Architect
+  links:
+    - icon: fa-twitter
+      url: https://twitter.com/donmandonguy
+    - icon: fa-linkedin
+      url: https://www.linkedin.com/in/donny-flynn-578149a4/
+  name: Donny Flynn
+  organization: Census
+doug_beatty:
+  image_url: /img/blog/authors/dbeatty.jpeg
+  job_title: Senior Developer Experience Advocate
+  name: Doug Beatty
+  organization: dbt Labs
+emily_riederer:
+  description: Emily is very smart.
```

Comment:
I didn't know you all lied on your developer blog! 

Shall we change this to the following?

> Emily Riederer is a Senior Analytics Manager at Capital One where she leads a teams delivering a portfolio of data products, enterprise analysis tools, and data science solutions to business partners. As part of the dbt community, she develops and maintains the `dbtplyr` package. In her spare time, Emily frequently writes about data (see emilyriederer.com, *The R Markdown Cookbook*, and *97 Things Every Data Engineering Should Know*), reviews technical manuscripts for CRC Press, and supports open research software engineering as a member of the rOpenSci editorial board.

Reply:
My mistake! Emily is very very smart 😉 

Applied your changes ⭐